### PR TITLE
SDK-280 : Add age verified helper

### PR DESCRIPTION
**Pull request by sidibos**
I will add a test case for this after this review if needed.

### CODE REVIEW

File: src/profile_service/index.js [hunk 1]
```diff
@@ -15,6 +16,8 @@ const ActivityDetails = function main(parsedResponse, decryptedProfile) {
     acc[propName] = current[propName];
     return acc;
   }, {});
+  const age = new Age(this.profile);
+  this.profile.isAgeVerified = age.isVerified();
```
Comment (vassyz) [inline src/profile_service/index.js]: Can't we add a conditional here `if (age.isVerified() !== null)`? It should not add the property if age verification does not exist. Adding this null property breaks the tests as well.

Comment (sidibos) [reply src/profile_service/index.js]: I have double checked I can see it's not adding the attribute to the profile if the value is not selected in the Dashboard - I could add an if there.

Comment (vassyz) [reply src/profile_service/index.js]: What do you mean by consistent? The other attributes are not being sent if they are null.

**Comment by vassyz** [comment]
Looks good, can we also update `browser.js` within this PR?

**Pull request merged**